tests/pipelines/test_pipelines_image_segmentation.py

```diff
@@ -383,12 +406,11 @@ def test_integration_torch_image_segmentation(self):
     @require_torch
     @slow
     def test_threshold(self):
+        self.maxDiff = None
```

Comment:
What does this line do?

If we're changing the state of the tester class, we probably want to set it back at the end to keep tests independent.

Reply:
Oops ! forgot to remove (this enables more output in th etests when it fails allowing to show exactly what is different within a nested datastructure)